README.md

```diff
@@ -31,9 +31,8 @@ project:
 
 ```js
 {
-    "minimum-stability": "dev",
     "require": {
-        "phly/phly-blog": "dev-master"
+        "phly/phly-blog": "^3.0"
```

Comment:
It'll be tagged as `3.0.0`, right?

Reply:
I never did a 2.0, so this would be 2.0 here. :)